src/main/java/com/puppycrawl/tools/checkstyle/DefaultLogger.java

```diff
@@ -159,30 +166,24 @@ public void addError(AuditEvent event) {
     @Override
     public void addException(AuditEvent event, Throwable throwable) {
         synchronized (errorWriter) {
-            final Violation addExceptionMessage = new Violation(1,
-                Definitions.CHECKSTYLE_BUNDLE, ADD_EXCEPTION_MESSAGE,
-                new String[] {event.getFileName()}, null,
-                Violation.class, null);
-            errorWriter.println(addExceptionMessage.getViolation());
+            final LocalizedMessage addExceptionMessage = new LocalizedMessage(
+                    ADD_EXCEPTION_MESSAGE, event.getFileName());
+            errorWriter.println(addExceptionMessage.getMessage());
```

Comment:
Please name it 'exceptionMessage'
Variables should be noun.

Reply:
Done